libraries/imageslider/src/main/java/com/trendyol/uicomponents/imageslider/ImageSliderViewState.kt

```diff
@@ -1,14 +1,29 @@
 package com.trendyol.uicomponents.imageslider
 
+import android.view.View
+
 data class ImageSliderViewState(
     val imageList: List<String>,
     var isImageDynamic: Boolean? = null,
-    var imageHeight: Int? = null
+    var imageHeight: Int? = null,
+    var isIndicatorAlwaysVisible: Boolean = false
 ) {
 
     fun isSingleImage() = imageList.size == 1
 
     fun isSingleImageVisible() = isSingleImage()
 
-    private fun isMultiImage() = imageList.size.greaterThan(1)
+    fun isMultiImage() = imageList.size.greaterThan(1)
+
+    fun getIndicatorVisibility(): Int {
```

Comment:
in the future we need to provide a data-binding adapter for this visibility feature. We have alread had this adapter inside our internal codebase

#futurenote :)